src/redux/actions/loadUser.ts

```diff
@@ -2,11 +2,32 @@ import {Dispatch} from "react";
 import AsyncStorage from "@react-native-async-storage/async-storage";
 import axios, {AxiosError} from "axios";
 import {URI} from "../../URI";
+import {userLoadFailed, userLoadRequest, userLoadSuccess} from "../types/types";
 
-export const loadUser = () => async (dispatch: Dispatch<any>) => {
+interface LoadUserRequestAction {
+    type: 'userLoadRequest';
```

Comment:
Here and in other places see https://redux.js.org/style-guide/#write-action-types-as-domaineventname for action type naming